File: java/src/main/java/com/experian/dvs/client/server/address/Address.java
package com.experian.dvs.client.server.address;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class Address {

    @JsonProperty("unspecified")
    private List<String> addressLines;

    public Address() {
    }

    public Address(final String singleline) {
        this(List.of(singleline));
    }

    public Address(final List<String> lines) {
        this.addressLines = lines;
    }

    public  List<String> getAddressLines() {
        return this.addressLines;
    }

    public void setAddressLines(final List<String> addressLines) {
        this.addressLines = addressLines;
    }
}

File: csharp/DVSClient/DVSClient/server/address/Address.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address
{
    public class Address
    {
        [JsonProperty("unspecified")]
        public IEnumerable<string>? AddressLines { get; set; }

        public Address()
        {
        }

        public Address(string singleline) : this(new List<string> { singleline })
        {
        }

        public Address(List<string> lines)
        {
            AddressLines = lines;
        }
    }
}